terraform/main.mk

```diff
@@ -82,6 +82,12 @@ terraform.output-to-ssm: ## Manual upload output.json to AWS SSM. Output.json en
 	@ cd $(ENV_DIR) && \
 	$(CMD_SAVE_OUTPUT_TO_SSM)
 
+terraform.plan-output: terraform.init ## Terraform plan output for Github Action
```

Comment:
I would just create a `terraform.plan` task and add it also as a dependency to `terraform.apply`.

Reply:
Fixed